developer-workflow/lang-changes.rst

```diff
@@ -69,20 +145,18 @@ see `Justifying Python Language Changes`_.
 PEP process
 -----------
 
-Once you are certain you have a language change proposal
-which will appeal to the general Python community,
-you can begin the :abbr:`PEP (Python enhancement proposal)` process
-to officially propose the change.
-See :pep:`1` for information on PEPs and the PEP process,
-and the :pep:`PEP Index <0>` for examples.
+Once you are certain you have a language change proposal which will appeal to
+the general Python community, you can begin the PEP
+process to officially propose the change.  See :pep:`1` for
+information on PEPs and the PEP process, and the :pep:`PEP Index <0>` for
+examples.
```

Comment:
Why change the wrapping here? Starting sentences/clauses on their own lines reduces future diffs, and can generally be a good idea.

Reply:
I can't tell what you are suggesting: 1) I should change the line breaks to correspond to clauses, or 2) I shouldn't change the wrapping from the text on main?   Some paragraphs have been re-wrapped already.